src/pages/org-dashboard/org-dashboard.jsx

```diff
@@ -255,6 +261,32 @@ function OrgDashboard() {
             </PopOut>
             )
         }
+        {
+            badgeFormOpened && (
```

Comment:
I think this one was accidentally duplicated, I will remove this also.

Reply:
Go for it